features/antidote_effect.feature

```diff
@@ -0,0 +1,27 @@
+        Scenario: Antidote effect implementation
+
+Feature: Antidote effect validation
+                Given I get target
+                When I should see the numerical target
+                And I should see target number should be in the range of citizens list
+                And I should see target is relevant to alive citizens
+                Then validation method should return True
+
+Feature: Antidote effect activation
+                Given activate method is called
+                Then active player should get message "Выберите гражданина, который следующий ход будет защищен от смертельной атаки..."
+                And active player should enter the target number
+                When validation is successful
+                Then target number should be set as effect target
+                When validation is failed
+                Then activation should be repeated
+
+Feature: Antidote effect resolve
+                Given resolve method is called and effect life should be more than 0
```

Comment:
What does 'effect life' means?

Reply:
Effect life it is variable that we use to check how long effect should be in status ACTIVATED and changed status to FINISHED